src/lib/annotations/__tests__/CommentBox-test.js

```diff
@@ -121,10 +129,22 @@ describe('lib/annotations/CommentBox', () => {
         it('should add the hidden class to the comment box element', () => {
             commentBox.hide();
```

Comment:
might be a good idea to expect commentbox.containerel.contains(CLASS_HIDDEN) to be false before calling commentbox.hide() as an extra check